### ISSUE TITLE
Undo shortcut for (de)selecting games

### ISSUE THREAD

**Issue by unrealed** [JosefNemec/Playnite#2035]
It would be great to be able to press a shortcut (probably Ctrl+Z) and then have it deselect or reselect any games that you have just (de)selected. Ideally, there would be a big internal log of many of the last (de)selections so we could undo many times in a row.

The reason I want this is that I have in the past wanted to try to properly maintain my Playnite database by adding data for which games I've played/beaten/completed/abandoned, etc. The problem I encountered happens when (since I have 1400+ games) I want to lots of bulk edits by selecting tons of games at once and making changes to all of them in one go. But when I select lots of games, I have in the past just clicked elsewhere or accidentally let go of Shift and just selecting one game, losing my whole huge selection. 

Having the ability to undo to reselect those games would mean I wouldn't actually be too scared of wasting tons of time by having these accidents to maintain my database.

**Comment by JosefNemec** [comment]
I don't think I ever saw an application to have undo for a list selection.

**Comment by darklinkpower** [comment]
There are applications where you can press Shift and click an already selected element to unselect it.

**Comment by JosefNemec** [comment]
That's how it works in Playnite as well, if you have selection of multiple items, then CTRL-Click removes an item from a selection.

**Comment by darklinkpower** [comment]
You are right. Considering that, I think that's a much better approach to this.

In my opinion Action Undo makes sense in command and keyboard based applications but not in something like Playnite. And if the user wants to select in bulk, the game list could first be filtered first and select everything with Ctrl+A and https://github.com/JosefNemec/Playnite/issues/1895 will help a lot for this in the future.
